src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -21,8 +21,10 @@
 #include <ie_ngraph_utils.hpp>
 
 #include <snippets/op/subgraph.hpp>
+#include "snippets/pass/matmul_to_brgemm.hpp"
 #include "emitters/cpu_generator.hpp"
 #include "utils/cpu_utils.hpp"
+#include "snippets_transformations/enforce_precision.hpp"
```

Comment:
As we discussed offline, we have to guarantee that `Subgraph` with `MatMul` inside and enabled `EnforceBF16` have `BF16` inputs. Honestly, I don't see this condition in the PR

You need to set `BF16` [on inputs](https://github.com/openvinotoolkit/openvino/blob/18c876bf2388bfd6d3ec48134e0c7a1ce3dad535/src/plugins/intel_cpu/src/nodes/subgraph.cpp#L183) in plugin `Subgraph` when we have `MatMul` and enabled `EnforceBF16`

Moreover, need to add comment why we do so

Reply:
I see that this comment is marked as resolved. I believe it was miss click because I don't see your answer or changes. If not, could you please elaborate why you marked as resolved?

Reply:
my bad, was done, thanks